src/Nut/DebugRoutes.php

```diff
@@ -36,9 +37,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $table = new Table($output);
         $table->setHeaders([
-            'Bind name',
-            'Path matching parameter',
-            'Method(s)',
+            [new TableCell('System routes', ['colspan' => 3])],
+            ['Bind Name', 'Path Match Pattern', 'Method(s)']
```

Comment:
`Bind Name` - Maybe just `Name`? You call `bind()` to set the _route name_, and "bind" isn't used anywhere else.

`Path Match Pattern` - Maybe just `Path`? Symfony's referred to the _path_ as [_pattern_](https://github.com/symfony/routing/blob/2.8/Route.php#L150), but it is deprecated.